util/src/main/java/com/google/cloud/hadoop/util/RetryHttpInitializerOptions.java

```diff
@@ -42,6 +44,9 @@ public static Builder builder() {
 
   public abstract Builder toBuilder();
 
+  /** Whether the RetryHttpInitializer is enabled or not. */
+  public abstract boolean getIsEnabled();
```

Comment:
rename to `isEnabled`

Reply:
the `<AutoValue>` requires that all methods either use `set/get` convention or none use. Please let me know if there's a workaround.